pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -283,6 +299,33 @@ func (gce *GCECloud) ProviderName() string {
 	return ProviderName
 }
 
+// Region returns the region
+func (gce *GCECloud) Region() string {
+	return gce.region
+}
+
+// OnXPN returns whether bool indicating whether the cluster is
+// running on a cross project network.
+func (gce *GCECloud) OnXPN() bool {
+	return gce.onXPN
+}
+
+// NetworkURL returns the network url
+func (gce *GCECloud) NetworkURL() string {
+	return gce.networkURL
+}
+
+// SubnetworkURL returns the subnetwork url
+func (gce *GCECloud) SubnetworkURL() string {
+	return gce.subnetworkURL
+}
+
+// NodeTags returns the list of node tags specified from the config file
+// If this list is empty, users should call GCECloud.ComputeHostTags
```

Comment:
Does it make sense to make the user have to call both methods to get at the tags? Why not populate the tags if they are missing?